pkg/telemetry/README.md

```diff
@@ -0,0 +1,313 @@
+# Telemetry Package
```

Comment:
I'd move all this content to a block comment in `telemetry.go` as this is the convention in Go.

Reply:
Love it